README.md

```diff
@@ -45,6 +46,14 @@ Version: 2.3.0 | [Changelog](CHANGELOG.md) | [Issues](https://github.com/bigblue
     ```bash
     $ docker-compose exec greenlight bundle exec rake admin:create
     ```
+7. If you want to build a different tag/branch or use another repo or fork:
```

Comment:
this appears like "step 7", but actually in the wrong order and for all the normal setups even unrelated.

maybe this could rather go into the development doc?